lib/apiary/command/preview.rb

```diff
@@ -116,17 +90,49 @@ def write_generated_path(path, outfile)
         File.write(outfile, File.read(path))
       end
 
-      def generate_static(path)
-        File.open(preview_path(path), "w") do |file|
-          file.write(query_apiary(@options.api_host, path))
+      def generate
+        template = load_preview_template()
+
+        data = {
+          title: File.basename(@options.path, ".*"),
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.